tools/test-proxy/scripts/transition-scripts/README.md

```diff
@@ -43,19 +73,27 @@ The location of the automatically restored assets is colloquially referred to as
 
 ## Running the script
 
-[generate-assets-json.ps1](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/transition-scripts/generate-assets-json.ps1) is a standalone powershell script with no supporting script requirements. The easiest way to run the script would be to pull the <https://github.com/Azure/azure-sdk-tools> repository locally and run the script directly from there.
+[generate-assets-json.ps1](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/transition-scripts/generate-assets-json.ps1) is a standalone powershell script with no supporting script requirements. The easiest way to run the script would be to use a one-liner [defined above](#download-the-transition-script-locally) to grab the file directly. **Please ensure you have the newest version of this script before continuing!**
 
 ```powershell
+# if downloading the file singly, cd to the directory containing generate-assets-json.ps1
 cd "<target-language-repo>/sdk/<service>"
-<language-repo-root>/tools/test-proxy/transition-scripts/generate-assets-json.ps1
+<path-to-transition-script>/generate-assets-json.ps1
 ```
 
-The script needs to be executed inside an `sdk/<ServiceDirectory>` or deeper, in an up to date language repository. A good rule here would be look at where the ci.yml is for an service directory. In the case where each library for a given service directory has their own pipelines, at the `sdk/<ServiceDirectory><Library>` level, it is recommended that the assets.json is created there. If the `ci.yml` exists deeper then the `sdk/<ServiceDirectory>/<Library>` level, then it is recommended to run the script from that directory.
+The script needs to be executed inside an `sdk/<ServiceDirectory>` or deeper and from within an up to date language repository. A good rule here would be look at where the ci.yml is for an service directory. In the case where each library for a given service directory has their own pipelines, at the `sdk/<ServiceDirectory><Library>` level, it is recommended that the assets.json is created there. If the `ci.yml` exists deeper then the `sdk/<ServiceDirectory>/<Library>` level, then it is recommended to run the script from that directory.
+
+```powershell
+# calling transition script against tool, given local clones of azure-sdk-for-java and azure-sdk-tools
+cd c:/src/azure-sdk-for-java/sdk/attestation
+<path-to-transition-script>/generate-assets-json.ps1 -TestProxyExe "test-proxy" -InitialPush
```

Comment:
I think you have good docs on how to override the exe above, so I would remove it here.

```
<path-to-transition-script>/generate-assets-json.ps1 -InitialPush
```